features/home_page/map.feature

```diff
@@ -60,3 +60,7 @@ Feature: Map of local charities
     Then the coordinates for "Harrow Bereavement Counselling" and "Youth UK" should be the same
     Then the coordinates for "Age UK" and "Youth UK" should not be the same
 
+  Scenario: Showing meaning of large map icons
```

Comment:
Might want to make this a scenario outline.  We have this legend appearing on more than just the home page -- including the individual organisation page.